examples/src/main/scala/com/example/payments/ExampleFlow.scala

```diff
@@ -89,12 +89,13 @@ class ExampleFlow(client: ZWorkflowClient) {
   )(transactionId:   UUID
   ) =
     workflowStub.signal(
-      ZSignal.signal(paymentWorkflow.confirmTransaction _)
-    )(
-      zinput(
-        ConfirmTransactionCommand(id = transactionId.toProto, confirmationCode = "42")
+      ZSignal.signal(
```

Comment:
This one looks much easier than the old ZSignal.
Planning to do the same for QueryMethod and WorkflowMethod